central/notifications/handler/handler_test.go

```diff
@@ -0,0 +1,86 @@
+package handler
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	dsMocks "github.com/stackrox/rox/central/notifications/datastore/mocks"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/notifications"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/mock/gomock"
+)
+
+func TestNotificationsHandler(t *testing.T) {
+	t.Parallel()
+	suite.Run(t, new(handlerTestSuite))
+}
+
+type handlerTestSuite struct {
+	suite.Suite
+	mockCtrl *gomock.Controller
+	mutex    sync.RWMutex
+
+	datastore          *dsMocks.MockDataStore
+	notificationStream notifications.Stream
+	handler            *handlerImpl
+}
+
+func (s *handlerTestSuite) SetupTest() {
+	s.mockCtrl = gomock.NewController(s.T())
+
+	s.datastore = dsMocks.NewMockDataStore(s.mockCtrl)
+	s.notificationStream = notifications.NewStream()
+	s.handler = newHandler(s.datastore, s.notificationStream).(*handlerImpl)
+	flushInterval = 10 * time.Millisecond
+	s.handler.Start()
+}
+
+func (s *handlerTestSuite) TearDownTest() {
+	s.mockCtrl.Finish()
```

Comment:
Same comment as in other PRs w.r.t [gomock.Controller.Finish](https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish) not being required